internal/api/phone.go

```diff
@@ -114,3 +111,13 @@ func (a *API) sendPhoneConfirmation(ctx context.Context, tx *storage.Connection,
 
 	return messageID, errors.Wrap(tx.UpdateOnly(user, includeFields...), "Database error updating user for confirmation")
 }
+
+func parseSmsTemplate(SMSTemplate *template.Template, otp string) (string, error) {
```

Comment:
Nit, maybe rename this to something more meaningful. It doesn't do parsing right now -- so just `generateSMSFromTemplate`?